tests/unit/GoFormGenerator/GoFormGroup.render-cases.js

```diff
@@ -0,0 +1,152 @@
+const basePropsData = {
+  value: 'my name',
+  field: {
+    inputType: 'text',
+    inputName: 'last_name',
+    placeholder: 'Your last name',
+    required: true,
+    validations: 'required'
+  }
+}
+
+const modelPropsData = {
+  value: 'my name',
+  field: {
+    inputType: 'text',
+    model: 'last_name',
+    inputName: 'last_name',
+    placeholder: 'Your last name',
+    required: true,
+    validations: 'required'
+  },
+}
+
+// render cases for computed hasErrors
+export const renderCaseHasErrors = [
+  [
+    'when there are no vee-validate and data base errors',
+    {
+      data () {
+        return {
+          displayFieldsErrors: false,
+        }
+      },
+      propsData: basePropsData,
+      computed: {
+        hasErrorVeeValidate: () => false,
+        fieldModelErrors: () => [],
+      }
+    },
+    false
+  ],
+  [
+    'when there are vee-validate errors',
+    {
+      data () {
+        return {
+          displayFieldsErrors: false,
+        }
+      },
+      propsData: basePropsData,
+      computed: {
+        hasErrorVeeValidate: () => true,
+        fieldModelErrors: () => [],
+      }
+    },
+    true
+  ],
+  [
+    'when there are database errors but display fields error is not activated',
```

Comment:
when there are database errors but the display of the fields error is not activated